public/Player.js

```diff
@@ -36,68 +39,33 @@ export default class Player {
     this.domElement = htmlPlayersContainer.querySelector(`#${this.name}`);
```

Comment:
Can't comment on the line but 2 lines above this there is still calls to the `document` instead of querying from the board entity, remember that querying from the `document` makes the query traverse the whole DOM tree which is not always the most efficient way